resources/english.yml

```diff
@@ -1025,4 +1025,7 @@ msg_neutral_towns: 'Neutral towns'
 
 # Added in 0.64
 msg_err_no_nation_with_that_name: 'No nation found with name %s'
-msg_err_cannot_unclaim_homeblock: 'You cannot unclaim your homeblock, use ''/town set homeblock'' to move your homeblock.'
\ No newline at end of file
+msg_err_cannot_unclaim_homeblock: 'You cannot unclaim your homeblock, use ''/town set homeblock'' to move your homeblock.'
+
+# Added in 0.67
+msg_warn_claim_plot_belongs_to_group: 'This plot belongs to a plot group, are you sure you want to buy it?'
```

Comment:
Should change it so it ends with ...are you sure you want to buy it for %s?' so that we can show the player the amount it costs.